rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.highlight/60_unified_matched_fields.yml

```diff
@@ -0,0 +1,82 @@
+setup:
+  - requires:
+      cluster_features: ["gte_v8.13.0"]
```

Comment:
This is a `gte`, so it would run on 8.13. I think you want `gte_v8.15.0`. I honestly don't know how these cluster_features versions things work.

Reply:
Resolved through the last changes.